core/system/rule_manager.go

```diff
@@ -55,9 +31,28 @@ func GetRules() []*SystemRule {
 }
 
 // LoadRules loads given system rules to the rule manager, while all previous rules will be replaced.
-func LoadRules(rules []*SystemRule) (bool, error) {
-	ruleChan <- rules
-	return true, nil
+func LoadRules(rules []*SystemRule) bool {
```

Comment:
It's better to pass up the error to the upper layer.

Reply:
In the current case, the meaning of the two return values is repeated. Ehen bool is true, error is always nil. Therefore I delete the error.
If we need pass up the error to the upper layer, we can delete the bool. What do you think of this way?

Reply:
Actually, the `bool` here was designed to indicate whether the internal map has been changed, not for the error.